plasmapy/physics/tests/test_dielectric.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for functions that calculate plasma dielectric parameters in
+dielectry.py"""
+
+import numpy as np
+from astropy import units as u
+
+from ..dielectric import (cold_plasma_permittivity_LRP,
+                          cold_plasma_permittivity_SDP)
+
+B = 1.0 * u.T
+n = [1e18/u.m**3]
+omega = 55e6*u.rad/u.s
+
+single_species = ['e']
+two_species = ['e', 'D+']
+three_species = ['e', 'D+', 'H+']
+
+
+def test_cold_plasma_permittivity_SDP():
```

Comment:
Make these class based otherwise it's tough to figure out what failed and why.
One assert per method and a docstring in each method which precisely describes the test.

Reply:
`test_distribution.py` is a good example if you need a template to work from.

Reply:
It's a good place for that, I kinda agree.